app/views/shared/_work_version_files.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render(EmbargoDetailComponent.new(work_version: work_version)) %>
+<%= render(FilesVisibilityDetailComponent.new(work_version: work_version)) %>
 <%= render(WithdrawnDetailComponent.new(work_version: work_version)) %>
```

Comment:
@awead - no worries! After talking with Seth in Slack on Friday, he preferred to keep it as two separate components and was OK with 2 alert boxes:

> Seth Erickson 9:50
> it might make sense to treat withdrawn in a separate component
> 
> Nick Tiberi  9:51 AM
> ok. the current implementation is that withdrawn and embargoed are 2 separate components, and both could possibly display at the same time. and there is no component for “penn state only”
> 
> Seth Erickson  9:59 AM
> right. so one component for penn state only & embargo, and separate component for withdrawn. This makes sense to me because they relate to slightly different things (works and versions, respectively)